src/greptimeai/extractor/openai_extractor/chat_completion_extractor.py

```diff
@@ -39,11 +43,13 @@ def pre_extract(self, *args, **kwargs) -> Extraction:
     @override
     def post_extract(self, resp: ChatCompletion) -> Extraction:
         extraction = super().post_extract(resp)
-
-        choices = extraction.event_attributes.get("choices", None)
-        if choices:
+        if resp.choices:
```

Comment:
we don't rely on resp after `post_extract`. What about change the signature of `parse_choices` and use Dict, instead of calling `model_dump()`?